src/panels/config/automation/condition/types/ha-automation-condition-time.ts

```diff
@@ -36,10 +62,80 @@ export class HaTimeCondition extends LitElement implements ConditionElement {
         .value=${before}
         @value-changed=${this._valueChanged}
       ></paper-input>
+
+      ${this._days.map(
+        (day) => html`
+          <ha-formfield
+            class="flex"
+            .label=${this.hass!.localize(
+              `ui.panel.config.automation.editor.conditions.type.time.weekdays.${day.name}`
+            )}
+            .dir=${computeRTLDirection(this.hass!)}
+            alignEnd
+            spaceBetween
```

Comment:
```suggestion
            alignEnd
            spaceBetween
            class="flex"
            .label=${this.hass!.localize(
              `ui.panel.config.automation.editor.conditions.type.time.weekdays.${day.name}`
            )}
            .dir=${computeRTLDirection(this.hass!)}
```

Just a code readability thing.